app/helpers/batch_helper.rb

```diff
@@ -98,52 +128,68 @@ def batch_raw_data_table(batch)
   # Temporary method helpful for debugging during development
   def batch_processed_data_table(batch)
     govuk_table(
-      caption: "Processed (#{batch.processed_rows.count} submissions)",
-      head: batch.processed_headers,
-      rows: batch.processed_rows
+      caption: "Processed submissions (#{batch.pending_induction_submissions.count} total)",
+      head: ['TRN', 'Date of birth', 'Status', 'Error messages'],
+      rows: batch.pending_induction_submissions.map do |submission|
+        [
+          submission.trn,
+          submission.date_of_birth&.to_fs(:govuk) || '-',
+          submission.error_messages.any? ? 'Error' : 'Valid',
+          submission.error_messages.join(', ').presence || '-'
+        ]
+      end
     )
   end
 
   # Temporary method helpful for debugging during development
   def batch_download_data_table(batch)
+    error_submissions = batch.pending_induction_submissions.with_errors
+
     govuk_table(
-      caption: "Downloadable bad CSV data (#{batch.failed_submissions.count} rows)",
-      head: batch.row_headings.values,
-      rows: batch.failed_submissions
+      caption: "Submissions with errors (#{error_submissions.count} rows)",
+      head: ['TRN', 'Date of birth', 'Error messages'],
+      rows: error_submissions.map do |submission|
+        [
+          submission.trn,
```

Comment:
Important distinction about your change, is when you get a row with an invalid TRN with 8 characters for example.

The TRN here will have been "sanitised" in order to be persisted, so the error won't match what the support agent sees.

The method name was chosen because it details what was available to the user as a downloadable CSV file, which the table doesn't do anymore. Now it basically repeats the table above, filtering out the successes. I think this is less useful for support, if not redundant.